src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java

```diff
@@ -0,0 +1,340 @@
+import org.apache.commons.io.FilenameUtils;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.QueryParameter;
+import org.kohsuke.stapler.StaplerRequest;
+import com.mathworks.ci.AddMatlabToPathBuildWrapper.MatlabBuildWrapperDescriptor;
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.Launcher.ProcStarter;
+import hudson.model.AbstractProject;
+import hudson.model.Result;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.tasks.BuildStepDescriptor;
+import hudson.tasks.Builder;
+import hudson.util.FormValidation;
+import hudson.util.FormValidation.Kind;
+import jenkins.model.Jenkins;
+import jenkins.tasks.SimpleBuildStep;
+import net.sf.json.JSONObject;
+
+public class RunMatlabTestsBuilder extends Builder implements SimpleBuildStep {
+    
+    private int buildResult;
+    private EnvVars env;
+    private boolean tapChkBx;
+    private boolean junitChkBx;
+    private boolean coberturaChkBx;
+    private boolean stmResultsChkBx;
+    private boolean modelCoverageChkBx;
+    private boolean pdfReportChkBx;
+
+    @DataBoundConstructor
+    public RunMatlabTestsBuilder() {
+
+
+    }
+
+
+    // Getter and Setters to access local members
+
+
+    @DataBoundSetter
+    public void setTapChkBx(boolean tapChkBx) {
+        this.tapChkBx = tapChkBx;
+    }
+
+    @DataBoundSetter
+    public void setJunitChkBx(boolean junitChkBx) {
+        this.junitChkBx = junitChkBx;
+    }
+
+    @DataBoundSetter
+    public void setCoberturaChkBx(boolean coberturaChkBx) {
+        this.coberturaChkBx = coberturaChkBx;
+    }
+    
+    @DataBoundSetter
+    public void setStmResultsChkBx(boolean stmResultsChkBx) {
+        this.stmResultsChkBx = stmResultsChkBx;
+    }
+    
+    @DataBoundSetter
+    public void setModelCoverageChkBx(boolean modelCoverageChkBx) {
+        this.modelCoverageChkBx = modelCoverageChkBx;
+    }
+    
+    @DataBoundSetter
+    public void setPdfReportChkBx(boolean pdfReportChkBx) {
+        this.pdfReportChkBx = pdfReportChkBx;
+    }
+            
+    public boolean getTapChkBx() {
+        return tapChkBx;
+    }
+
+    public boolean getJunitChkBx() {
+        return junitChkBx;
+    }
+
+    public boolean getCoberturaChkBx() {
+        return coberturaChkBx;
+    }
+
+    public boolean getStmResultsChkBx() {
+        return stmResultsChkBx;
+    }
+            
+    public boolean getModelCoverageChkBx() {
+        return modelCoverageChkBx;
+    }
+    
+    public boolean getPdfReportChkBx() {
+        return pdfReportChkBx;
+    }
+    
+    private void setEnv(EnvVars env) {
+        this.env = env;
+    }
+    
+    private EnvVars getEnv() {
+        return this.env;
+    }
+    
+    @Symbol("RunMatlabTests")
+    @Extension
+    public static class MatlabTestDescriptor extends BuildStepDescriptor<Builder> {

```

Comment:
Descriptor class name needs to be updated to match the new Wrapper name.

Reply:
Done.